Assignment3/protocol.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, init_shared_key):
         # This is how messsages are sent by default if we do not have a unique session key
         # Established
         # This is stored as bytes
-        self._init_shared_key = self._prepKeyForAES(init_shared_key.encode('utf-8'))
+        self._init_shared_key = init_shared_key
```

Comment:
we can just set it to none and remove the init_shared_key paramater from the function.